ga4gh/backend_exceptions.py

```diff
@@ -0,0 +1,30 @@
+"""
+Exceptions for the backend (i.e. that aren't returned to clients)
```

Comment:
This isn't strictly true, is it? We do return a `CallSetNotInVariantSetException` to the client, so they know what happened.